etc/config-template.toml

```diff
@@ -55,9 +55,9 @@
 # also should less than total cpu cores.
 # scheduler-worker-pool-size = 4
 
-# A write flow control mechanism. When concurrent running write count over
+# A write flow control mechanism. When concurrent running write kvs count over
 # this threshold, will return too busy error.
```

Comment:
-> When the number of concurrent running writes exceeds this threshold, the "too busy" (or the exact error message content) error is displayed.